docs/content/en/docs/getting-started/orchestrate/_index.md

```diff
@@ -0,0 +1,6 @@
+---
+title: Getting started with the Lifecycle Toolkit
```

Comment:
This has the same title as the previous section. Maybe we can call this `Deployment Orchestration`. This can be done in a follow up :) 
@StackScribe